server/constants/index.js

```diff
@@ -0,0 +1,3 @@
+export const HEART_OF_AZEROTH = {
```

Comment:
IMO it would better fit here: https://github.com/WoWAnalyzer/WoWAnalyzer/tree/master/src/common/ITEMS/bfa

Reply:
Ah, I wasn't sure if /server was meant to remain localized to its own directory. 
Do you have a filename suggestion? I feel like the item is unique enough to not fall under any general category :/

Reply:
This will cause an import out of /server such as `import heartOfAzeroth from '../../../src/common/ITEMS/bfa/heartOfAzeroth';`, which will prevent server from being extracted out of this repository, are we sure this is something we want to do? 

I don't see any other instances of this happening in /server

Reply:
ye thinking about it, it does make sense to keep it here.
If Zer0 has anything against it, it should be easy enough to change anyway.

I'd say readd the whitespaces please and I'll merge it.

Reply:
Great, done